AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -98,6 +98,9 @@ @interface ASCollectionView () <ASRangeControllerDelegate, ASDataControllerSourc
   ASDataController *_dataController;
   ASRangeController *_rangeController;
   ASFlowLayoutController *_layoutController;
+
+  BOOL _batchUpdateFlag;
```

Comment:
instead of maintaining a flag, could we just check whether `_batchUpdateBlocks.count > 0`?  if not, can we rename this to something like `_performingBatchUpdates` so it's clearer what is being flagged?